net.cachapa.tudo.yaml

```diff
@@ -0,0 +1,24 @@
+id: net.cachapa.tudo
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: tudo
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+#  - --socket=pulseaudio
+  - --share=network
+modules:
+  - name: tudo
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - ./build-flatpak.sh
+    sources:
+      - type: dir
+        path: ../../build/linux/x64/release/bundle/
+      - type: dir
+        path: assets
```

Comment:
SO how does this work? You should not use dir sources, it should be a direct link to the source code/tarball/binary etc.

Reply:
To be honest this is my first time generating a Flatpak so I'm not sure if my approach is correct.

In the first case, the directory contains the executable + precompiled libs required for the application to run.

In the second the assets directory contains metadata for the Flatpak: build script, icon, .desktop, etc.

Reply:
> the directory contains the executable

And how is said directory found? We cannot go and grab it from your PC nor we can ship your PC.

It has to be a link.